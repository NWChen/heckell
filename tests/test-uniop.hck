let foo: int;
foo = -4;
print(foo);