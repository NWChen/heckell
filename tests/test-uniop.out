let foo: int
foo = -4
